linalg/lapack.hpp

```diff
@@ -13,8 +13,7 @@
 #define MFEM_LAPACK_HPP
 
 #include "../config/config.hpp"
-
-#ifdef MFEM_USE_LAPACK
+#include <complex>
```

Comment:
Should this be inside `#ifdef MFEM_USE_LAPACK`, or is it always needed?

Reply:
Moved it inside in c690c25058dd34c0ed9fbf21dd8c3dbf600a7755.